staff/maryam-malek/cv/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+   
+</head>
+<body>
+        <header>
+            <h1 id="name">Richard House</h1>
+            <address>
+                <p>Email adress:<br><a href="mailto:webmaster@example.com">webmaster@example.com</a></p>
+                <p>Portafolio<br><a href="mailto:webmaster@example.com">example.com</a></p>
+                <p>Street beta</p>
+                <p>London</p>
+            </address>
+        </header>  
+        <section id="studies">
```

Comment:
The id is not necessary, you must remove it

Reply:
ids are not a solution to semantics, but the correct elements themselves. do not use ids for no explicit reason.